learn2track/models/gru_mixture.py

```diff
@@ -77,6 +77,15 @@ def _get_mixture_samples(self, srng, mixture_weights, means, stds):
 
         return samples
 
+    def _get_max_component_samples(self, mixture_weights, means, stds):
+        batch_size = mixture_weights.shape[0]
+        xs = T.arange(0, batch_size)
+
+        choices = T.argmax(mixture_weights, axis=1)
+        samples = means[xs, choices]
+
+        return samples
+
     def seq_next(self, x_t, subject_ids=None):
```

Comment:
While at it, should we remove the `seq_next`, `seq_reset` methods? Or is it better to do it in another PR?